jenkins/create_state_county_country_basemap

```diff
@@ -0,0 +1,70 @@
+pipeline {
+  agent {
+        node {
+            label 'team:makerspace'
```

Comment:
This is the extra large slave, right?  Does this need it?

Reply:
k, i have just been in the habit of using it, but no it doesn't need it. ill use the less powerful slave.

Reply:
it works fine on the general slave!